text/trace/0157-messaging-semantic-conventions.md

```diff
@@ -0,0 +1,184 @@
+# Scenarios for Tracing semantic conventions for messaging
+
+This document aims to capture scenarios and a road map, both of which will
+serve as a basis for [stabilizing](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#stable)
+the [existing semantic conventions for messaging](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md),
+which are currently in an [experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#experimental)
+state. The goal is to declare messaging semantic conventions stable before the
+end of 2021.
+
+## Motivation
+
+Many observability scenarios involve messaging systems. For Distributed Tracing
+to be useful across the entire scenario, having good observability for
+messaging operations is critical. To achieve this, OpenTelemetry must provide
+stable conventions and guidelines for instrumenting messaging systems.
+
+Bringing the existing experimental semantic conventions for messaging to a
+stable state is a crucial step for users and instrumentation authors, as it
+allows them to rely on [stability guarantees](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#not-defined-semantic-conventions-stability),
+and thus to ship and use stable instrumentation.
+
+## Roadmap
+
+| Description | Done By     |
+|-------------|-------------|
+| This OTEP, consisting of scenarios and a proposed roadmap, is approved and merged. | 09/30/2021 |
+| [Stability guarantees](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#not-defined-semantic-conventions-stability) for semantic conventions are approved and merged. This is not strictly related to semantic conventions for messaging but is a prerequisite for stabilizing any semantic conventions. | 09/30/2021 |
+| An OTEP proposing a set of attributes and conventions covering the scenarios in this document is approved and merged. | 10/29/2021 |
+| Proposed specification changes are verified by prototypes for the scenarios and examples below. | 11/15/2021 |
+| The [specification for messaging semantic conventions for tracing](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md) are updated according to the OTEP mentioned above and are declared [stable](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#stable). | 11/30/2021 |
+
+## Terminology
+
+### Message
+
+A "message" is a transport wrapper for the transfer of information between two
+or more parties. The information is a combination of data and metadata.
```

Comment:
Can we say it's between two parties? It's possible for messages to be sent to a destination and never read again. Or read at some far off point in the future when there was no concept of what party would receive the message

Reply:
I removed the "between to or more parties".